ui/imports/shared/views/AssetsView.qml

```diff
@@ -108,16 +108,28 @@ ColumnLayout {
                 FastExpressionRole {
                     name: "currentCurrencyBalance"
                     expression: {
-                        let totalBalance = d.getTotalBalance(model.balances, model.decimals, root.addressFilters, root.networkFilters)
```

Comment:
This turns out to be the culprit? Afaik I warned exactly against this :)

Reply:
actually no its doesnt solve the problem, I guess i'll test with Michals fast sorters and filters and see if it improves